### PR TITLE
chore(examples): add mobile design for media demo

### PR DESCRIPTION
**Pull request by eunjae-lee**
This adds mobile design for the media demo.

## Preview
[See here](https://deploy-preview-3943--instantsearchjs.netlify.com/examples/media/)

## Related
- [The parent PR](https://github.com/algolia/instantsearch.js/pull/3915)
- [Design](https://app.zeplin.io/project/5d1393093588af695d431e9e/screen/5d1393af1ab3c4042c7a14d5)

### CODE REVIEW

File: examples/media/index.html [hunk 1]
```diff
@@ -44,7 +45,44 @@
       <div class="header-content">
         <div class="searchbox-container">
           <label for="search-input">
-            <svg class="magnifier-icon" viewBox="0 0 17 17">
+            <svg
+              data-layout="mobile"
+              class="magnifier-icon"
+              xmlns="http://www.w3.org/2000/svg"
```
Comment (francoischalifour) [inline examples/media/index.html]: `xmlns` (and `xmlns:xlink`) attributes are not required in HTML I think (I learned that recently).

Comment (eunjae-lee) [reply examples/media/index.html]: oh right, https://stackoverflow.com/a/18468348/11593190
good to know!

Comment (eunjae-lee) [reply examples/media/index.html]: Parcel failed to build it without `xmlns` :(

**Comment by eunjae-lee** [comment]
@francoischalifour About the categories and locations being broken on the desktop, I think it has something to do with the mobile widgets.
When I comment out `categoriesMobile` from `search.ts`, the categories on the desktop start showing correctly.
Do you have any clue on this?

**Comment by francoischalifour** [comment]
@eunjae-lee It could be because they use the same panel?

**Comment by eunjae-lee** [comment]
@francoischalifour oh you're right. Thanks!

**Pull request merged**